relayer/processor/path_end_runtime.go

```diff
@@ -839,3 +850,27 @@ func (pathEnd *pathEndRuntime) trackProcessingMessage(tracker messageToTrack) ui
 
 	return retryCount
 }
+
+func (pathEnd *pathEndRuntime) localhostSentinelProofPacket(
```

Comment:
Are these two sentinel methods called anywhere?

Reply:
Yes, they are used in the message assembly logic in place of the regular connection and packet related proofs used on non-localhost connections.